modules/nixos/globals.nix

```
@@ -1,9 +1,13 @@
-{ lib, ... }: {
+{ lib, config, ... }: {
   options.catppuccin = {
     flavour = lib.mkOption {
       type = lib.ctp.types.flavourOption;
       default = "latte";
       description = "Global Catppuccin flavour";
     };
+
+    enable = (lib.mkEnableOption "Catppuccin globally") // {
+      default = config.catppuccin.enable;
+    };
```

```suggestion
    enable = lib.mkEnableOption "Catppuccin globally";
```
pretty sure this is also infinite recursion :p

---

It is definitely infinite recursion :P